plugins/telegram-resources/src/components/Chat.svelte

```diff
@@ -82,7 +82,7 @@
         updateAccountsQuery(accountsIds)
       },
       {
-        sort: { sendOn: SortingOrder.Descending },
+        sort: { sendOn: SortingOrder.Ascending },
```

Comment:
You should get last 500 messages, not first. It should be `Descending`, just use `messages = res.reverse()` in 77